### ISSUE TITLE
[Android] Auto-Logging Purchase Issue after Upgrading to Play Billing Library 5

### ISSUE THREAD

**Issue by RafalWilkowski** [facebook/facebook-sdk-for-unity#700]
## Checklist

- [x] I've updated to the latest released version of the SDK
- [x] I've searched for existing [GitHub issues](https://github.com/facebook/facebook-sdk-for-unity/issues)
- [x] I've looked for existing answers on [Stack Overflow](https://facebook.stackoverflow.com), the [Facebook Developer Community Forum](https://developers.facebook.com/community/) and the [Facebook Developers Group](https://www.facebook.com/groups/fbdevelopers)
- [x] I've read the [Code of Conduct](CODE_OF_CONDUCT.md)
- [x] This issue is not security related and can safely be disclosed publicly on GitHub

## Environment

Describe your dev environment here, giving as many details as possible. If you have them, make sure to include:

- Unity Editor Version: `2021.3.26f1`
- Unity SDK Version: `16.0.0`
- Installation Platform & Version: `Android` version `any`

## Goals

What do you want to achieve?

Autolog purchases on Android.

What do you expect to happen?

See purchase events in FB dashboard.

What actually happened? Can you provide a stack trace?

No purchase events in FB dashboard.

What are the steps necessary to reproduce this issue?

Upgrade Unity IAP to version with PBL 5 (e.g. IAP 4.8.0) and autolog enabled.

The problem appears to be within the Facebook Android SDK, which still utilizes the queryPurchases method removed in PBL5.

**Comment by JamesCBarry** [comment]
I'm running into this exact same issue which appeared after I updated IAP to 4.10.0 to comply with the new Play Billing Library. Still have not found a fix.

**Comment by benyamangeria** [comment]
Hi, did you manage to resolve the issue?

**Comment by JamesCBarry** [comment]
No. I had to set up manual tracking for all purchases instead of relying on auto-tracking.

**Comment by kristofgiber** [comment]
Still having the same issue. Strange that it hasn't been fixed more than a year later. I've seen a lot of others posting about it on Meta forums and Stackoverflow.
